From 0ef3b76150f01e9610102c48eb8f1629ddf7f655 Mon Sep 17 00:00:00 2001
From: zhang sanshan <b51434@freescale.com>
Date: Fri, 26 Dec 2014 14:26:47 +0800
Subject: [PATCH 31/34] MA-5998 change MKUBIFS tool

the old mkfs.ubifs is from host server.
This patch  change set mkfs_ubifs as the default tool.

Signed-off-by: zhang sanshan <b51434@freescale.com>
---
 core/config.mk                    | 2 +-
 tools/releasetools/build_image.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/config.mk b/core/config.mk
index 01c33e5..d6aa6f7 100644
--- a/core/config.mk
+++ b/core/config.mk
@@ -408,7 +408,7 @@ else
 MKBOOTIMG := $(BOARD_CUSTOM_MKBOOTIMG)
 endif
 MKYAFFS2 := $(HOST_OUT_EXECUTABLES)/mkyaffs2image$(HOST_EXECUTABLE_SUFFIX)
-MKUBIFS := /usr/sbin/mkfs.ubifs
+MKUBIFS := $(HOST_OUT_EXECUTABLES)/mkfs_ubifs$(HOST_EXECUTABLE_SUFFIX) 
 MKUBIRAW := $(HOST_OUT_EXECUTABLES)/ubinize$(HOST_EXECUTABLE_SUFFIX)
 APICHECK := $(HOST_OUT_EXECUTABLES)/apicheck$(HOST_EXECUTABLE_SUFFIX)
 FS_GET_STATS := $(HOST_OUT_EXECUTABLES)/fs_get_stats$(HOST_EXECUTABLE_SUFFIX)
diff --git a/tools/releasetools/build_image.py b/tools/releasetools/build_image.py
index 003613b..9704d79 100755
--- a/tools/releasetools/build_image.py
+++ b/tools/releasetools/build_image.py
@@ -299,7 +299,7 @@ def BuildImage(in_dir, prop_dict, out_file, target_out=None):
     build_command.extend([out_file, prop_dict["partition_size"]])
   elif fs_type.startswith("ubifs"):
     # add ubifs image generate support.
-    build_command = ["mkfs.ubifs", "-d"]
+    build_command = ["mkfs_ubifs", "-d"]
     build_command.append(in_dir)
     build_command.append("-o")
     build_command.append(out_file)
-- 
1.8.0

